File: Aula02/Java/vetor2.java
public class vetor2{
    public static void main(String[] args){
        Scanner scaner = new Scanner(System.in);

        int[] numbers = new int[6];
        int sumEven = 0;
        int countOdd = 0;

        System.out.println("Digite 6 números inteiros: " );

        for ( int i = 0; i< 6; i++){
            numbers[i] = scanner.nextInt();
        }


        System.out.println("Números pares digitados: " );
        for( int num : numbers){
            if(num % 2 == 0){
                System.out.println(num + " ");
                sumEven += num; 
                
            }
        }


        System.out.println("Soma dos números pares: " + sumEven);

        System.out.println("Números ímpares digitados: ");
        for (int num : numbers){
            if(num % 2 != 0){
                System.out.println(num + );

            }
        }

        System.out.println("Quantidades de números ímpares: "+countOdd);
    }
}

File: Aula02/GO/vetor2.go
package main

import (
	"fmt"
)

func main(){

	var numbers [6]int
	var sumEven int
	var countOdd int

	fmt.Printf("Digite 6 números inteiros: ")

	for i := 0; i<6; i++{
		fmt.Scan(&numbers[i])
	} 

	fmt.Printf("Números pares digitados: ")
	for _, num := range numbers {
		if num%2 ==0{
			fmt.Printf(num, " ")
			sumEven += num
		}
	}
	fmt.Printf()

	fmt.Printf("Soma dos números pares:" , sumEven)

	fmt.Printf("Números ímpares digitados: ")
	for _, num := range numbers {
		if num%2 !=0{
			fmt.Printf(num, " ")
			countOdd++
		}
	}

	fmt.Printf("Quantidade de números ímpares: ", countOdd)
}